src/View/layout.html.twig

```diff
@@ -7,7 +7,7 @@
 		<link rel="icon" href="/assets/images/icons/favicon.png">
 		<link rel="stylesheet" href="/assets/css/style.css">
 		<link rel="stylesheet" href="/assets/css/header.css">
-    	<link rel="stylesheet" href="/assets/css/footer.css">
+    <link rel="stylesheet" href="/assets/css/footer.css">
         <link rel="stylesheet" href="/assets/css/reset.css">
         <link rel="stylesheet" href="/assets/css/contact.css">
```

Comment:
contac and legal should'nt be inside layout , but on stylesheet blocks in their own template page